### PR TITLE
chore: edit readme

### PR DESCRIPTION
**Pull request by Im-Kal**
Hello!

I have fixed some of the minor annoyances of the current README and also made the screenshot's background transparent.

### CODE REVIEW

**Comment by Im-Kal** [comment]
image
Image attached above.

File: README.md [hunk 1]
```diff
@@ -3,14 +3,17 @@
...
 </p>
 
+---
+
+
 Command-line tool to customize the official Spotify client.
 Supports Windows, MacOS and Linux.
 
-<img src="https://user-images.githubusercontent.com/26436809/118751529-d0abcf00-b8a4-11eb-9876-8b15f930a691.png" alt="img" align="right" width="500px">  
+<img src="https://i.imgur.com/VUXGKYZ.png" alt="img" align="right" width="560px" height="400px">
```
Comment (theRealPadster) [inline README.md:16]: Maybe we should upload the image to the github repo, in case the link goes down in the future?

Comment (Im-Kal) [reply README.md:16]: Yeah sure, do upload it in the repo if you have the means to

Comment (theRealPadster) [reply README.md:16]: You can do it in this PR

Comment (Im-Kal) [reply README.md:16]: In my repo?

Comment (Im-Kal) [reply README.md:16]: Done doing it. /.github/assets/logo.png

Comment (Im-Kal) [reply README.md]: Added to README.

**Review by afonsojramos — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -19,7 +22,10 @@ Supports Windows, MacOS and Linux.
...
-#### [Installation](https://spicetify.app/docs/getting-started)
-#### [Basic Usage](https://spicetify.app/docs/getting-started#basic-usage)
-#### [Customization](https://spicetify.app/docs/development/customization)
-#### [FAQ](https://spicetify.app/docs/faq)
+
+### Links
+- [Installation](https://spicetify.app/docs/getting-started)
+- [Basic Usage](https://spicetify.app/docs/getting-started#basic-usage)
+- [Customization](https://spicetify.app/docs/development/customization)
+- [FAQ](https://spicetify.app/docs/faq)
```
Comment (afonsojramos) [inline README.md:30]: Honestly, I don't like the bullet points here. Can you rollback?

**Review by afonsojramos — CHANGES_REQUESTED** [review]

**Review by afonsojramos — APPROVED** [review]

**Pull request merged**